### DISCUSSION TITLE
Is personal identifiable safe?

### DISCUSSION THREAD

**Discussion by blanca-perez** [jessecambon/tidygeo#149] [Q&A]
(I posted this question in Issues, but I think now it's better categorized here)

I want to use TidyGeocoder to get latitude and longitude of addresses of individuals. Given that's personal identifiable data, I'm searching for a geocoding method which doesn't retain this data. Looking at TidyGeocoder, I see that one of the servicer options is the Census Geocoder: https://www.census.gov/programs-surveys/geography/technical-documentation/complete-technical-documentation/census-geocoder.html

This is an statement from the Census Geocoder that describes the data this geocoder retains for both API and web interface and for what reasons:

"When you submit a geocode request, the web server automatically collects certain technical information from your computer and about your connection. The only information that is stored is IP Address (for batch and single address submissions) and submitted address (only for single address submissions).
The information is stored on a server, which is internal to the Census network. This server is only accessible by Census Bureau staff, who are bound by the confidentiality requirements set forth in Title 13 of the United States Code, and requires multiple levels of approval.

This information is only used to monitor and track the performance of the Geocoder."

I want to check if the use of TidyGeocoder to obtain latitude and longitude from the Census Geocoder meets the needs for keeping data safe.

**Comment by jessecambon** [comment]
Hi @blanca-perez, according to that excerpt you've posted it sounds like the US Census retains the addresses, but keeps them confidential. If you want more details you could give them an email (it's at the bottom of that page you linked to and I've found them to be pretty responsive). 

This might not exactly fit your use case, but the Geocodio service has a [HIPAA compliant endpoint](https://www.geocod.io/healthcare/). You can enable the use of this endpoint by using `api_options = list(geocodio_hipaa = TRUE)`.
